### ISSUE TITLE
Consider breaking into pieces

### ISSUE THREAD

**Issue by crstauf** [crstauf/query-monitor-extend#22]
Rather than one plugin with multiple enhancements and new panels, break each off into its own directory, for use as an mu-plugin.

Example:
```
qm-extend-constants
- qmx-constants-collector.php
- qmx-constants-output.php
```

This might not work for all current additions, so carefully evaluate before starting.

**Comment by crstauf** [comment]
Unless someone expresses a reason to not do this, will be merged into master soon.
